common/src/main/java/org/tron/common/prometheus/MetricsGauge.java

```diff
@@ -33,6 +33,8 @@ class MetricsGauge {
         LABEL_STAKE_VERSION, LABEL_RESOURCE_TYPE);
     init(RESOURCE_WINDOW_SIZE, "tron resource window size.",
         ACCOUNT_ADDRESS, LABEL_RESOURCE_TYPE);
+
+    init(MetricKeys.Gauge.VERIFY_SIGN_SIZE, "tron verify sign trx size.", "type");
```

Comment:
What types are there? So far there seems to be only one.

Reply:
yeah , it's only trxs type.  thanks , bobo !